certbot/cli.py

```diff
@@ -1213,8 +1213,16 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         " run if an attempt was made to obtain/renew a certificate. If"
         " multiple renewed certificates have identical post-hooks, only"
         " one will be run.")
-    helpful.add("renew", "--renew-hook",
-                action=_RenewHookAction, help=argparse.SUPPRESS)
+    helpful.add(
```

Comment:
Looks like there's a bunch of code cleaning going on in this PR, love seeing those! Let's put those in a separate PR to make tracing the history of code changes easier on us in the future.

Reply:
looks like this one got missed

Reply:
I'm excited to fix the formatting as well! When someone goes to trace the history of this line's changes, they will get confused if they see this commit, which seems like it might be related, but in fact isn't. If it's in a different PR, it will be clear that they need to go one more back, and since there's an upcoming cleanup PR anyway, might as well group it with those.

Reply:
All right, refactor reverted. But I will take my revenge on theses files with my PEP8 machine gun in a future PR ^^

Reply:
I cannot wait